proto/executor.proto

```diff
@@ -23,6 +23,7 @@ message ExecuteRequest {
 message ExecuteContext {
 	bool isInteractivitySupported = 1;
 	bytes slackState = 2;
+	string kubeConfig = 3;
```

Comment:
Remember that kubeconfig should be also passed to Source plugins as a part of the source plugin process launch.

So: 
for executor - generate & pass kubeconfig per command execution
for source - generate & pass kubeconfig before starting the source plugin binary